.prettierrc

```diff
@@ -6,5 +6,6 @@
   "trailingComma": "all",
   "bracketSpacing": true,
   "jsxBracketSameLine": false,
-  "arrowParens": "always"
-}
\ No newline at end of file
+  "arrowParens": "always",
+  "endOfLine": "auto"
```

Comment:
I think that we want `"endOfLine"` to be `LF` to be consistent with unix machines

Reply:
I agree, we absolutely need consistent endOfLine. You can see this setting also here: https://github.com/HospitalRun/hospitalrun-frontend/blob/master/.editorconfig#L9